modules/weko-schema-ui/weko_schema_ui/utils.py

```diff
@@ -24,11 +24,13 @@
 import re
 from functools import reduce
 
+from flask import current_app, request
```

Comment:
Unused import statement `from flask import current_app, request`.
Please remove it.